test/pensions/migrations.unit.spec.js

```diff
@@ -0,0 +1,100 @@
+import { expect } from 'chai';
+
+import migrations from '../../src/js/pensions/migrations';
+
+describe.only('Pension migrations', () => {
+  it('should set url to service history if validation error', () => {
+    const { formData, metadata } = migrations[0]({
+      formData: {
+        servicePeriods: [{
+          activeServiceDateRange: {
+            from: '2012-01-01'
+          }
+        }],
+        veteranDateOfBirth: '2013-01-01'
+      },
+      metadata: {
+        returnUrl: 'asdf'
+      }
+    });
+
+    expect(metadata.returnUrl).to.equal('/military/history');
+    expect(formData).to.be.an('object');
```

Comment:
Out of curiosity, why check for this?

Reply:
I want to make sure we don't accidentally forget to return `formData`.